reference/global-api-information.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Global API Information
+---
+
+It is possible to provide some global API information in the `ApiDefinition.define` block with the `info` method. You may define this metadata for all versions of your API, or only for a specific version. This is purely informational output in the JSON files, it is *not* used in the doc browser or anywhere else yet.
```

Comment:
before the "This is purely informational..." , It might be nice to add something like:

All definitions at the global level (i.e. without specifying a version) will be inherited by all versions. Any directive defined within a version will overwrite anything inherited.